src/ds/d500/d500-private.cpp

```diff
@@ -222,13 +229,16 @@ namespace librealsense
             intrinsics.height = height;
 
             // For D555e, model will be brown and we need the unrectified intrinsics
-            auto rect_params = compute_rect_params_from_resolution( table->rgb_coefficients_table.distortion_model
-                                                                            == RS2_DISTORTION_BROWN_CONRADY
-                                                                        ? table->rgb_coefficients_table.base_instrinsics
-                                                                        : table->rectified_intrinsics,
-                                                                    width,
-                                                                    height,
-                                                                    false );  // symmetry not needed for RGB
+            bool use_base_intrinsics
+                = ( table->rgb_coefficients_table.distortion_model == RS2_DISTORTION_BROWN_CONRADY
+                    && table->rgb_coefficients_table.reserved[3] == 0 );
```

Comment:
Maybe add a comment that we use `reserved[3]` as a flag here to indicate the full distortion module is not really brown but we treat it as such because the HW fixes fisheye distortion.

Reply:
Added